### PR TITLE
Announce Trusted Setup V4 ceremony

### PR DESCRIPTION
**Pull request by 0xjei**
## Description
This PR adds the announcement of the MPC Phase 2 Trusted Setup ceremony for Semaphore V4 circuits variants (from 1 up to 32).

## Related Issue(s)
see #546 

## Other information
- The information about the precise dates is __NOT__ present. Therefore, I kindly ask to review styles/aspect and do __NOT__ merge until confirmation of dates!
- A new custom reusable `Banner` component has been added to the `website` app. Minor changes have been made to avoid `Banner`-`NavBar` overlap.

## Checklist

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation
-   [x] My changes generate no new warnings
-   [x] I have run `yarn format` and `yarn lint` without getting any errors
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [x] New and existing unit tests pass locally with my changes
-   [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: apps/website/src/app/layout.tsx [hunk 1]
```diff
@@ -30,9 +31,21 @@ export default function RootLayout({ children }: { children: React.ReactNode })
         <html lang="en" suppressHydrationWarning>
             <body suppressHydrationWarning>
                 <Providers>
+                    <Banner>
+                        Semaphore V4
+                        <Link
```
Comment (cedoor) [inline apps/website/src/app/layout.tsx:36]: The hover effect on links should be the same as the other links on the website. I think there should just be a change in the underline. You can check the other links.

**Review by cedoor — CHANGES_REQUESTED** [review]
LGTM! Just left a comment about CSS

File: apps/website/src/app/layout.tsx (hunk 1 above)
Comment (0xjei) [reply apps/website/src/app/layout.tsx:36]: correct - we should make `primary.600` the underline and not the text itself.

Comment (0xjei) [reply apps/website/src/app/layout.tsx:36]: updated!

**Review by cedoor — APPROVED** [review]

File: apps/website/src/app/layout.tsx (hunk 1 above)
Comment (cedoor) [reply apps/website/src/app/layout.tsx:36]: Great! It's ready then

File: apps/docs/versioned_docs/version-V4-beta/trusted-setup.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+sidebar_position: 6
+---
+
+# Trusted Setup
+
+gm, everyone 🕶️ we are excited to announce the upcoming Multi-Party Computation (MPC) Phase 2 Trusted Setup ceremony for the Semaphore V4 circuit - this is crucial for establishing a robust security foundation for the cryptographic protocol 🎉
```
Comment (vplasencia) [inline apps/docs/versioned_docs/version-V4-beta/trusted-setup.md]: The glasses can't be seen with dark mode. Should we use another emoji or remove it?

File: apps/docs/versioned_docs/version-V4-beta/trusted-setup.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+The ceremony will take place from **June 10** to **July 10**. If all goes to plan, finalization should take place on **July 15** with the announcement of the final beacon on **July 12**.
```
Comment (vplasencia) [inline apps/docs/versioned_docs/version-V4-beta/trusted-setup.md:9]: Should we update these dates? Should it be Jun 11?

Comment (0xjei) [reply apps/docs/versioned_docs/version-V4-beta/trusted-setup.md:9]: officially is June 10th - it's open from yesterday but delayed the setup :( TL;DR it's correct from a config standpoint

File: apps/docs/versioned_docs/version-V4-beta/trusted-setup.md (hunk 2 above)
Comment (0xjei) [reply apps/docs/versioned_docs/version-V4-beta/trusted-setup.md]: you are right, we can get rid of them! xD

**Review by vplasencia — APPROVED** [review]

**Pull request merged**